common/src/main/java/com/microsoft/identity/common/internal/result/MsalBrokerResultAdapter.java

```diff
@@ -196,6 +189,54 @@ public BaseException getBaseExceptionFromBundle(@NonNull final Bundle resultBund
 
     }
 
+    public Bundle bundleFromBrokerResult(final BrokerResult brokerResult){
+        final Bundle resultBundle = new Bundle();
+        final String brokerResultString = sRequestAdapterGsonInstance.toJson(
+                brokerResult,
+                BrokerResult.class
+        );
+        try {
+            byte[] compressedBytes = compressString(brokerResultString);
+            Logger.info(TAG, "Broker Result, raw payload size:"
+                    + brokerResultString.getBytes().length + " ,compressed bytes " + compressedBytes.length
+            );
+            resultBundle.putByteArray(
+                    AuthenticationConstants.Broker.BROKER_RESULT_V2_COMPRESSED,
+                    compressedBytes
+            );
+        }catch (IOException e){
+            Logger.error(TAG, "Failed to compress Broker Result, sending as jsonString ", e);
```

Comment:
Let's do two logs here: one for `error` to indicate that compression failed, and then a second as `warn` to re-iterate that and indicate that we are going to be sending jsonString

Reply:
I don't see the necessity to add two logs here , given we can convey the message using one and theoretically we should never hit this